test/connection.test.js

```diff
@@ -1,17 +1,42 @@
 require('./init.js');
 var assert = require('assert');
+var DataSource = require('loopback-datasource-juggler').DataSource;
+var mysqlConnector = require('../');
+var URL = require('url');
```

Comment:
Nitpick: we usually use the same property name as is the module name, i.e. lowercase `url`.